lib/command/run.rb

```diff
@@ -335,6 +290,18 @@ def build_job_start_yaml # rubocop:disable Metrics/CyclomaticComplexity, Metrics
         job_start_hash["args"].push('eval "$CPL_RUNNER_SCRIPT"')
       end
 
+      image = config.options[:image]
+      image_link = if image
+                     image = latest_image if image == "latest"
+                     "/org/#{config.org}/image/#{image}"
+                   else
+                     container_spec["image"]
```

Comment:
What is the purpose of setting this if it's the same? Some race condition?

Reply:
Renamed it to `original_container_spec` so it's clearer: 84361d349c7c9d55acb924772e3d427df9da5132